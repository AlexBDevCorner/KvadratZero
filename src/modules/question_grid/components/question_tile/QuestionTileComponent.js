import React from 'react'

export default ({ id, text, events, points }) => (
  <div>
    <div className="question-text">{text}</div>
    <div className="question-points">{points}</div>
  </div>
)
