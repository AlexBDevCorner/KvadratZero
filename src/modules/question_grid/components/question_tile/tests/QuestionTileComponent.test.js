import React from 'react'
import { shallow } from 'enzyme'

import QuestionTileComponent from '../QuestionTileComponent'

const props = { text : "question text", points : 0 }
const component = shallow(<QuestionTileComponent { ...props } />)

describe('QuestionTileComponent tests :', () => {

  it('should properly render component', () => {
    expect(component).toMatchSnapshot()
  })

  it('should render question text', () => {
    const text = component.find('.question-text').text()
    expect(text).toBe("question text")
  })

  it('should render question points', () => {
    const points = component.find('.question-points').text()
    expect(points).toBe("0")
  })


})
