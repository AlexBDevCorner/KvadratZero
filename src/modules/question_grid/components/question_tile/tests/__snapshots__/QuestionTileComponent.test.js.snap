// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionTileComponent tests : should properly render component 1`] = `
<div>
  <div
    className="question-text"
  >
    question text
  </div>
  <div
    className="question-points"
  >
    0
  </div>
</div>
`;
